tests/link/pytorch/test_basic.py

```diff
@@ -343,3 +380,44 @@ def test_pytorch_OpFromGraph():
 
     f = FunctionGraph([x, y, z], [out])
     compare_pytorch_and_py(f, [xv, yv, zv])
+
+
+def test_ScalarLoop_Elemwise():
+    n_steps = int64("n_steps")
+    x0 = float64("x0")
+    x = x0 * 2
+    until = x >= 10
+
+    scalarop = ScalarLoop(init=[x0], update=[x], until=until)
+    op = Elemwise(scalarop)
+
+    n_steps = pt.scalar("n_steps", dtype="int32")
+    x0 = pt.vector("x0", dtype="float32")
```

Comment:
Add a second carry of type `tensor(shape=(7, 3, 1)` so it broadcasts with the vector x0.

This will make sure multiple carry are working and we are getting the right shape outputs.

Reply:
done